les/protocol.go

```diff
@@ -43,7 +43,7 @@ var (
 )
 
 // Number of implemented message corresponding to different protocol versions.
-var ProtocolLengths = map[uint]uint64{lpv2: 24, lpv3: 26}
+var ProtocolLengths = map[uint]uint64{lpv2: 24, lpv3: 28}
```

Comment:
We may need to define `lpv4` to maintain backward compatibility but I am not quite sure.